include/deal.II/meshworker/scratch_data.h

```diff
@@ -1016,8 +1246,7 @@ namespace MeshWorker
     const std::string &global_vector_name,
     Number             dummy) const
   {
-    const unsigned int n_dofs =
-      get_current_fe_values().get_fe().n_dofs_per_cell();
+    const unsigned int n_dofs = local_dof_indices.size();
```

Comment:
This was a bug when it came to evaluating on interfaces. The number of DoFs stored is now variable and interface specific, so without knowing which FEValues-like object was last reinit'd, only the vector itself knows about the number of local DoFs.  `ScratchData<dim, spacedim>::extract_local_dof_values()`  already contains the correct logic, so it need only be corrected here.